### PR TITLE
[feature] :support alibaba Ai

### PR DESCRIPTION
**Pull request by Yanshuming1**
[feature] :support alibaba Ai

### CODE REVIEW

**Review by zqr10159 — APPROVED** [review]

File: manager/src/main/resources/sureness.yml [hunk 1]
```diff
@@ -63,6 +63,7 @@ resourceRole:
 # rule: api===method 
 # eg: /api/v1/source3===get means /api/v1/source3===get can be access by anyone, no need auth.
 excludedResource:
+  - /api/ai/**===*
```
Comment (tomsun28) [inline manager/src/main/resources/sureness.yml:66]: hi, can not add this api in excludeResource when prod env

File: manager/src/main/java/org/apache/hertzbeat/manager/controller/MonitorsController.java [hunk 2]
```diff
@@ -160,7 +160,7 @@ public ResponseEntity<Message<Void>> deleteMonitors(
             @Parameter(description = "Monitoring ID List", example = "6565463543") @RequestParam(required = false) List<Long> ids
     ) {
         if (ids != null && !ids.isEmpty()) {
-            monitorService.deleteMonitors(new HashSet<>(ids));
+            monitorService.cancelManageMonitors(new HashSet<>(ids));
```
Comment (tomsun28) [inline manager/src/main/java/org/apache/hertzbeat/manager/controller/MonitorsController.java:163]: this maybe wrong mix?

Comment (Yanshuming1) [reply manager/src/main/java/org/apache/hertzbeat/manager/controller/MonitorsController.java:163]: 这个代码不小心push了， 可以在合并时候选择不合并吗？ 或者我重新merge一下

File: manager/src/main/resources/sureness.yml (hunk 1 above)
Comment (Yanshuming1) [reply manager/src/main/resources/sureness.yml:66]: 好的，稍后我删掉

**Review by zqr10159 — APPROVED** [review]

**Comment by tomsun28** [comment]
image

has this merge the wrong?

**Pull request merged**